example/spec/parallel/orca/OrcaBackupAndRecoverySpec.js

```diff
@@ -133,19 +139,13 @@ describe('The S3 Ingest Granules workflow', () => {
 
   afterAll(async () => {
     if (!isOrcaIncluded) return;
-
-    // clean up stack state added by test
-    await removePublishedGranule({
+    await removeCollectionAndAllDependencies({
       prefix: config.stackName,
-      granuleId: inputPayload.granules[0].granuleId,
+      collection,
     });
+
     await Promise.all([
       deleteFolder(config.bucket, testDataFolder),
-      deleteCollection({
```

Comment:
This was failing as the async trigger eventually triggers a recovery workflow, but we don't know what it is in the test.   Rather than poll ES, opted to use the full collection wipe helper to take out the granule/executions as we *should* be sure that anything referenced via postgres table references is related to this test only.